ui/raidboss/data/05-shb/alliance/the_copied_factory.js

```diff
@@ -110,5 +236,111 @@
         en: 'Back and Sides',
       },
     },
+    {
+      id: 'Copied Engels Marx Crush',
+      regex: / 14:4746:Engels starts using Marx Crush/,
+      infoText: {
+        en: 'Kill Claws',
+      },
+    },
+    {
+      id: 'Copied Engels Precision Guided Missile',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:00C6:/,
+      alertText: function(data, matches) {
+        if (matches[1] == data.me) {
+          return {
+            en: 'Tank Buster on YOU',
+            de: 'Tankbuster auf DIR',
+            fr: 'Tankbuster sur VOUS',
+          };
+        }
+        if (data.role == 'healer') {
```

Comment:
Sorry to be picky, but can you make this infoText for consistency? I think I just don't like being so noisy for healers on alliance tankbusters.